wzdx/standard_to_enhanced/navjoy_translator.py

```diff
@@ -0,0 +1,252 @@
+import argparse
+import copy
+import json
+import logging
+from collections import OrderedDict
```

Comment:
Can we clean up these imports? Looks like the following aren't used:
- OrderedDict
- datetime
- array_tools
- polygon_tools